elfi/methods/inference/bolfire.py

```diff
@@ -58,10 +58,9 @@ def __init__(self,
             custom target_model is given.
         n_initial_evidence: int, optional
             Number of initial evidence.
-        acq_noise_var: float or np.ndarray, optional
+        acq_noise_var : float or dict, optional
             Variance(s) of the noise added in the default LCBSC acquisition method.
-            If an array, should be 1d specifying the variance for each dimension.
-        exploration_rate: float, optional
+            If a dictionary, values should be float specifying the variance for each dimension.        exploration_rate: float, optional
```

Comment:
exploration rate should start a new line

Reply:
of course - fixed